### PR TITLE
docs: modify the docs directory

### PR DESCRIPTION
**Pull request by guoqqqi**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] New feature provided

___
### New feature or improvement
- Describe the details and related test reports.

Changed the directory structure under docs.

### CODE REVIEW

File: docs/en/0.1.0/config.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "version": "0.1.0",
+  "sidebar": {}
+}
```
Comment (juzhiyuan) [inline docs/en/0.1.0/config.json:4]: new line

File: docs/en/0.1.0/config.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (juzhiyuan) [inline docs/en/0.1.0/config.json:1]: Only need the latest folder

File: docs/en/latest/ack.md [hunk 3]
```diff
@@ -71,4 +71,4 @@ helm install apisix-ingress-controller ./charts/apisix-ingress-controller \
 
 Change the `image.tag` to the apisix-ingress-controller version that you desire. You have to Wait for while until the correspdoning pods are running.
 
-Try to create some [resources](../CRD-specification.md) to verify the running status. As a minimalist example, see [proxy-the-httpbin-service](../samples/proxy-the-httpbin-service.md) to learn how to apply resources to drive the apisix-ingress-controller.
+Try to create some [resources](./CRD-specification.md) to verify the running status. As a minimalist example, see [proxy-the-httpbin-service](./proxy-the-httpbin-service.md) to learn how to apply resources to drive the apisix-ingress-controller.
```
Comment (juzhiyuan) [inline docs/en/latest/ack.md:74]: no, keep `/docs/en/latest/deployments/ack.md`

**Review by juzhiyuan — CHANGES_REQUESTED** [review]
Please check other changes

**Comment by guoqqqi** [comment]
image
The file was not found, is it `development.md`?

**Review by juzhiyuan — APPROVED** [review]

**Review by gxthrj — APPROVED** [review]
Great~

**Pull request merged**